allennlp/sanity_checks/batch_norm_verification.py

```diff
@@ -0,0 +1,85 @@
+"""
+Code based almost entirely on
+https://github.com/awaelchli/pytorch-lightning-snippets/commit/7db53f774715d635c59ef56f21a17634d246b2c5
+"""
+
+import torch
+from torch import nn as nn
+from typing import Tuple, List, Callable
+from allennlp.sanity_checks.verification_base import VerificationBase
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BatchNormVerification(VerificationBase):
+
+    normalization_layer = (
+        nn.BatchNorm1d,
+        nn.BatchNorm2d,
+        nn.BatchNorm3d,
+        nn.SyncBatchNorm,
+        nn.InstanceNorm1d,
+        nn.InstanceNorm2d,
+        nn.InstanceNorm3d,
+        nn.GroupNorm,
+        nn.LayerNorm,
```

Comment:
Given that it detects all of these, can we call it something else? `NormalizationVerification` or something?